draft-ietf-rtcweb-mdns-ice-candidates.md

```diff
@@ -140,6 +140,8 @@ For any host ICE candidate gathered by a browsing context as part of {{RFC8445}}
 
 7. Replace the IP address of the ICE host candidate with the name with ".local" appended to it. Expose the candidate.
 
+ICE host candidates generated using this procedure SHOULD have an unique mDNS name generated for each host candidate IP address, specifically an ICE agent using an interface with both IPv4 and IPv6 addresses SHOULD expose a different mDNS name for each address.
```

Comment:
Should this be MUST-strength? I think we probably want to take a position here.

Reply:
Changed to MUST.